slycot/tests/test_tb05ad.py

```diff
@@ -187,6 +197,29 @@ def test_tb05ad_resonance(self):
             transform.tb05ad(2, 1, 1, jomega, A, B, C, job='NH')
         assert cm.exception.info == 2
 
+    def test_tb05ad_balance(self):
+        """Specifically check for the balancing output.
+
+        Based on https://rdrr.io/rforge/expm/man/balance.html
```

Comment:
According to that link, this case has `SCALE=[1 1 1 3]`, `ILO=2, IHI=3` and thus only swaps 4 with 3 and 1 with 1. Does not really check the correct order of transformations done, does it?

To check the newly corrected transformation in TB05AD, `Bt`, `Ct` need to be checked for correct order as well.

Reply:
I created a new test, based on a random matrix, tweaked until that one showed permutation

Reply:
Could this be pinned to a fixed found matrix? Having random data in tests that could fail at times and does not at other times will make debugging hard.

Reply:
we're still covered by the random seed at the start of the file

Reply:
Ah okay, I didn't see that.